contrib/opencensus-ext-ocagent/setup.py

```diff
@@ -46,6 +46,6 @@
     license='Apache-2.0',
     packages=find_packages(exclude=('tests',)),
     namespace_packages=[],
-    url='https://github.com/census-instrumentation/opencensus-python/tree/master/contrib/opencensus-ext-ocagent',
+    url='https://github.com/census-instrumentation/opencensus-python/tree/master/contrib/opencensus-ext-ocagent',  # noqa: E501
```

Comment:
Why all these changes in this PR? Can they be made in a separate PR?

Reply:
This should go away after rebase.